orion/pkg/tests_integration/announce_integration_test.go

```diff
@@ -11,7 +11,7 @@ import (
 
 // Test: Create 3 Announcements and GetAll()
 func Test_CreateAnnouncements(t *testing.T) {
-	resetTable(t, domains.TABLE_ANNOUNCEMENTS)
+	resetAllTables(t)
```

Comment:
I find it odd we have to reset ALL tables for announcement tests... are you sure we have to do that?
It would be great to reset only announcements table in these tests since these tests shouldn't affect any other integration test.
If it does for some reason, it looks like we have an underlying bug somewhere!

I think we can keep resetTable() but you can move it to setupTests.go